pymeasure/instruments/hp/hp856Xx.py

```diff
@@ -748,6 +748,11 @@ def set_crt_adjustment_pattern(self):
         set_process=lambda v: str(v).upper()
     )
 
+    def write(self, command, **kwargs):
+        if "{amplitude_unit}" in command:
+            command = command.format(amplitude_unit=self.amplitude_unit)
+        super(HP856Xx, self).write(command, **kwargs)
```

Comment:
If you use `super`, you should not mention the current class in order to be more flexible:
```suggestion
        super().write(command, **kwargs)
```